server/views/partials/analytics/layout.njk

```diff
@@ -19,8 +19,7 @@
       text: "Protected characteristics",
       href: "/analytics/protected-characteristics",
       active: graphPageId == "protected-characteristics",
-      attributes: { "data-qa": "protected-characteristics" },
-      itemToRemove: not featureFlags.showPcAnalytics
+      attributes: { "data-qa": "protected-characteristics" }
     }
   ] | rejectattr('itemToRemove') %}
```

Comment:
don't need to filter the list anymore. you could even pass the list of items straight into the sub-nav macro

Reply:
Fair enough 👌